src/app/machines/StepCoreConfiguration.machine.ts

```diff
@@ -95,7 +96,13 @@ export const basicMachine = model.createMachine(
           },
           confirm: {
             target: '#done',
-            cond: 'isBasicConfigured',
+            actions: sendParent(({ name, sACreated }) => ({
+              analyticsEventName: `${CORE_CONFIGURATION} next`,
+              connectorName: name,
+              wasServiceAccountCreated: sACreated ? 'yes' : 'no',
+              type: 'sendAnalytics',
+            })),
```

Comment:
This is a typical analytics event that a step can send.  I opted to have each step send these vs trying to do it all from the CreateConnectorWizard machine.  The important field on these events for the state machine is the `type`, and for analytics the `analyticsEventName` is going to what's used when invoking our analytics backend.  Eventually :smile: